presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcDeleteDeltaPageSource.java

```diff
@@ -104,14 +102,15 @@ public OrcDeleteDeltaPageSource(
             Map<String, OrcColumn> acidColumns = uniqueIndex(
                     reader.getRootColumn().getNestedColumns(),
                     orcColumn -> orcColumn.getColumnName().toLowerCase(ENGLISH));
-            List<OrcColumn> rowIdColumns = ImmutableList.of(
+
+            List<OrcColumn> rowIdColumns;
+            rowIdColumns = ImmutableList.of(
                     acidColumns.get(ACID_COLUMN_ORIGINAL_TRANSACTION.toLowerCase(ENGLISH)),
-                    acidColumns.get(ACID_COLUMN_BUCKET.toLowerCase(ENGLISH)),
```

Comment:
why remove?
RowId still contains the bucket number (which is OK), so shouldn't this be retained here?

Reply:
We need <originalTransaction, bucketId, rowId> to correctly figure out if a row is deleted. 

In case of original files, we don't have ACID columns. We calculate rowId as given in OriginalFilesUtils, originalTrasaction is always 0. But, we don't know the bucketId. So, for faster read and simplification, we can drop not reading bucketId at all in this PR. That's why, I have dropped this Bucket column read code. Please let me know your take on this.